database/models/reaction_species.py

```diff
@@ -188,7 +188,23 @@ def to_rmg(self):
         )
 
     def __str__(self):
-        return "{s.id}".format(s=self)
+        stoich_reactants = []
+        stoich_products = []
+        for stoich, species in self.stoich_species():
+            if stoich < 0:
+                stoich_reactants.append((stoich, species))
+            else:
+                stoich_products.append((stoich, species))
+        left_side = " + ".join(
+            f"{int(abs(stoich)) if abs(stoich) != 1 else ''}{species.formula}"
+            for stoich, species in stoich_reactants
+        )
+        right_side = " + ".join(
+            f"{int(stoich) if stoich != 1 else ''}{species.formula}"
+            for stoich, species in stoich_products
+        )
+
+        return f"{left_side} -> {right_side}"
```

Comment:
I can't tell where this is used but at some point and is some contexts we might care about writing reversible reactions as <=> and irreversible ones as ->

Reply:
4c824da